rtfscluster/rtfs_cluster.go

```diff
@@ -105,7 +106,8 @@ func (cm *ClusterManager) RemovePinFromCluster(cidString string) error {
 // FetchLocalStatus is used to fetch the local status of all pins
 func (cm *ClusterManager) FetchLocalStatus() (map[gocid.Cid]string, error) {
 	var response = make(map[gocid.Cid]string)
-	pinInfo, err := cm.Client.StatusAll(true)
+	// todo: doublecheck if this is the right api.TrackerStatus to use
+	pinInfo, err := cm.Client.StatusAll(api.TrackerStatusPinned, true)
```

Comment:
same here (similar to https://github.com/RTradeLtd/Temporal/pull/287/files#r246644893)